### DISCUSSION TITLE
when play in IPAD or IPone These Tie symbols are not displayed But print

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#1135] [Q&A]
it like the img

![2](https://user-images.githubusercontent.com/17884664/228813402-01ef3fdb-0205-4d92-abc0-f10ba1eed2c6.png)

But when print it, they can show it again like thie img

![1](https://user-images.githubusercontent.com/17884664/228813486-28773b96-a120-4cb6-9518-776328b2cdcc.png)

the gp file
[24.zip](https://github.com/CoderLine/alphaTab/files/11110632/24.zip)

 ：
 alphatab  V1.3   ,Operating environment :  javascript web

**Comment by Danielku15** [comment]
I fear I don't understand you really. I know there is quite a language barrier here, but maybe you can try to formulate a bit more detailed what you are doing and what you expect? 

Looking at the file I can see that there is a tie on the first bar which is correctly displayed by alphaTab. But when you test on iOS (iPhone and iPad - what versions?) you cannot see them?

**Reply by goodgame365** [reply]
yes， when you open the https://www.alphatab.net/ ,It's happen on iPhone and iPad （Real machine），It actually happened on Android phones ，

Very strange, often occurring in the first line of the section（bar）

If you can't see it, please refresh the next page and the problem will arise 

Note: play on LayoutMode.Page;

My Ipone system is IOS 16.4



https://user-images.githubusercontent.com/17884664/229256845-dc33fb5d-04a6-4ca9-ab16-7ccec84ffd7f.mp4

**Reply by Danielku15** [reply]
I can reproduce the problem but cannot judge yet why this is happening. Will have to dig deeper here. It seems the tie is really fully missing and not drawn on purpose. But as this is only happening on iPad indicates a bug.

**Reply by goodgame365** [reply]
thank very much,Already fixed

**Comment by zhouhao27** [comment]
One question. You load tab locally or from server?

**Reply by Danielku15** [reply]
Please open own dedicated questions according to your needs and try to explain your overall setup.
